Automattic/jetpack
# WPCOM API: do not delete user when they have active WPCOM subs

Backend part of the fix for https://github.com/Automattic/wp-calypso/issues/27481

Mimics the behaviour we already have for domains - but extends it to all paid upgrades.

Differential Revision: D59343-code

This commit syncs r223557-wpcom.

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Not much to test here. This code is already wrapped in a `IS_WPCOM` check, so should be safe to merge from Jetpack point of view, as it should not be called at all in that context.
